api/sample.env

```diff
@@ -14,4 +14,8 @@ CSM_EMAIL_APP_USERNAME=
 CSM_EMAIL_APP_PASSWORD=
 CSM_EMAILS_TO_SEND_TO=
 CSM_TARGET_SMTP_SERVER_ADDRESS=
-CSM_TARGET_SMTP_SERVER_PORT=
\ No newline at end of file
+CSM_TARGET_SMTP_SERVER_PORT=
+
+# Error Email Contact Configuration
+CLOUD_RUN_LOGS_LINK=https://console.cloud.google.com/run/detail/us-central1/rsu-manager-cloud-run-api/logs?authuser=1&project=cdot-oim-cv-dev

```

Comment:
Lets name this CLOUD_LOGS_LINK or something. We don't plan to run this in GCP Cloud Run for much longer and have been trying to get rid of dependencies to GCP. This isn't really a dependency to Cloud Run to my understanding so it would be good to not make it sound like that.

Reply:
Updated to LOGS_LINK